bindata/network/multus/multus.yaml

```diff
@@ -45,6 +45,14 @@ spec:
               ;;
               rhel) rhelmajor=$(echo "${VERSION_ID}" | cut -f 1 -d .) 
               ;;
+              fedora)
+                if [ "${VARIANT_ID}" == "coreos" ]; then
+                  rhelmajor=8
```

Comment:
This will eventually become problematic when FCOS moves on a lot farther from RHEL8 - say a new openssl or whatever.  But for now seems sane.